rules/cloud/aws/cloudtrail/aws_iam_user_with_console_access_login_without_mfa.yml

```diff
@@ -0,0 +1,25 @@
+title: AWS IAM User with Console Access Login Without MFA
+id: 77caf516-34e5-4df9-b4db-20744fea0a60
+status: experimental
+description: |
+    Identifies AWS IAM user authentication to the AWS Management Console without Multi-Factor Authentication (MFA).
+    IAM users who can access the Console but haven't enabled MFA pose a security risk. These accounts could be compromised through credential stuffing, brute force attacks, or social engineering techniques.
+references:
+    - https://securitylabs.datadoghq.com/cloud-security-atlas/vulnerabilities/iam-user-without-mfa/
+    - https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference-aws-console-sign-in-events.html
+author: Thuya@Hacktilizer
+date: 2025-07-18
+tags:
+    - attack.initial-access
+    - attack.t1078.004
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection:
+        eventName: 'ConsoleLogin'
+        additionalEventData.MFAUsed: 'NO'
+    condition: selection
+falsepositives:
+    - Legit user login without MFA (Account is not using MFA for legitimate reason)
```

Comment:
```suggestion
    - Unlikely
```
a legitimate user logging in without MFA wouldn't be a false positive. It would be a legitimate alert that is accepted by the organization. e.g. the users that are allowed to log in like that should be filtered.